### ISSUE TITLE
Setting iro.ui.Box height property

### ISSUE THREAD

**Issue by michaelfarrelly** [jaames/iro.js#142]
Working on a custom project where the `height` of the Box picker needs to be distinct from `width`. But the current Box Dimensions only allows setting the `width` property.

example:
![image](https://user-images.githubusercontent.com/24235823/100022170-f184e300-2d9f-11eb-9458-84074644b6b7.png)

Would it be possible to have the following change? 

`iro-core`:
https://github.com/irojs/iro-core/blob/typescript/src/box.ts#L22-L29
```typescript
/**
 * @desc Get the bounding dimensions of the box
 * @param props - box props
 */
export function getBoxDimensions(props: Partial<BoxOptions>) {
  const { width, padding, handleRadius } = props;
  return {
    width: width,
    height: width,
    radius: padding + handleRadius
  };
}
```
Suggested:
```typescript
/**
 * @desc Get the bounding dimensions of the box
 * @param props - box props
 */
export function getBoxDimensions(props: Partial<BoxOptions>) {
  const { height, width, padding, handleRadius } = props;
  return {
    width: width,
    height: height ?? width,
    radius: padding + handleRadius
  };
}
```

**Comment by jaames** [comment]
Sure, I'll add it to the next release. The property name will have to be `boxHeight`, though

**Comment by jaames** [comment]
should be fixed in the latest release (5.3.0)
